🐛 No longer exists, https://centos7.iuscommunity.org/ius-release.rpm

## Description

Let's install python3.6 without using https://centos7.iuscommunity.org/ius-release.rpm

これも

```
Building wheel for mod-wsgi (setup.py) ... error
ERROR: Command errored out with exit status 1:
 command: /env3.6/bin/python3.6 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-5rn5m1i7/mod-wsgi_d376604334d544ba8c487596e52134b4/setup.py'"'"'; __file__='"'"'/tmp/pip-install-5rn5m1i7/mod-wsgi_d376604334d544ba8c487596e52134b4/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-veaayjmf
     cwd: /tmp/pip-install-5rn5m1i7/mod-wsgi_d376604334d544ba8c487596e52134b4/
Complete output (36 lines):
/usr/lib64/python3.6/distutils/dist.py:261: UserWarning: Unknown distribution option: 'bugtrack_url'
  warnings.warn(msg)
running bdist_wheel
running build
running build_py
creating build
creating build/lib.linux-x86_64-3.6
creating build/lib.linux-x86_64-3.6/mod_wsgi
copying src/__init__.py -> build/lib.linux-x86_64-3.6/mod_wsgi
creating build/lib.linux-x86_64-3.6/mod_wsgi/server
copying src/server/__init__.py -> build/lib.linux-x86_64-3.6/mod_wsgi/server
copying src/server/environ.py -> build/lib.linux-x86_64-3.6/mod_wsgi/server
copying src/server/apxs_config.py -> build/lib.linux-x86_64-3.6/mod_wsgi/server
creating build/lib.linux-x86_64-3.6/mod_wsgi/server/management
copying src/server/management/__init__.py -> build/lib.linux-x86_64-3.6/mod_wsgi/server/management
creating build/lib.linux-x86_64-3.6/mod_wsgi/server/management/commands
copying src/server/management/commands/runmodwsgi.py -> build/lib.linux-x86_64-3.6/mod_wsgi/server/management/commands
copying src/server/management/commands/__init__.py -> build/lib.linux-x86_64-3.6/mod_wsgi/server/management/commands
creating build/lib.linux-x86_64-3.6/mod_wsgi/docs
copying docs/_build/html/__init__.py -> build/lib.linux-x86_64-3.6/mod_wsgi/docs
creating build/lib.linux-x86_64-3.6/mod_wsgi/images
copying images/__init__.py -> build/lib.linux-x86_64-3.6/mod_wsgi/images
copying images/snake-whiskey.jpg -> build/lib.linux-x86_64-3.6/mod_wsgi/images
running build_ext
building 'mod_wsgi.server.mod_wsgi-py36' extension
creating build/temp.linux-x86_64-3.6
creating build/temp.linux-x86_64-3.6/src
creating build/temp.linux-x86_64-3.6/src/server
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include/httpd -I/env3.6/include -I/usr/include/python3.6m -c src/server/wsgi_thread.c -o build/temp.linux-x86_64-3.6/src/server/wsgi_thread.o -I/usr/include/httpd -I. -I/usr/include/apr-1 -DLINUX -D_REENTRANT -D_GNU_SOURCE -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -pthread -I/usr/include/apr-1 -I/usr/include/apr-1
In file included from src/server/wsgi_thread.h:24:0,
                 from src/server/wsgi_thread.c:21:
src/server/wsgi_python.h:24:20: fatal error: Python.h: No such file or directory
 #include <Python.h>
                    ^
compilation terminated.
error: command 'gcc' failed with exit status 1
----------------------------------------
ERROR: Failed building wheel for mod-wsgi
```

--- a/vagrant-provision/provision.sh
+++ b/vagrant-provision/provision.sh
@@ -119,3 +119,9 @@ sudo apachectl restart
 
 echo '----- Auto start -----'
 sudo systemctl enable httpd.service
+
+# axes を追加したことで、パーミッションの変更が必要になりました。
+# NOTE: これを設定しないと internal server error が発生します。
+#       ちなみにそのときのログは sudo cat /var/log/httpd/error_log で参照できます。
+sudo chmod 666 /var/log/mrrhp-apache/django-axes.log
+sudo chmod 666 /var/log/mrrhp-apache/error.log

--- a/vagrant-provision/provision.sh
+++ b/vagrant-provision/provision.sh
@@ -46,8 +46,8 @@ GRANT ALL PRIVILEGES ON *.* TO root@'192.168.33.1' IDENTIFIED BY 'password';
 __EOF__
 
 echo '----- Install Python -----'
-sudo yum install -y https://centos7.iuscommunity.org/ius-release.rpm
-sudo yum install -y python36u python36u-libs python36u-devel
+# python3-devel がないと mod_wsgi が入らない。
+sudo yum install python3 python3-devel -y
 which python
 which python3.6
 

